catalog.html

```diff
@@ -73,100 +73,145 @@ <h1 class="page_title">Магазин готовых шаблонов</h1>
-                  <input
-                    class="filter_input filter_input_radio"
-                    type="radio"
-                    name="grid"
-                    id="adaptive-grid"
-                    checked
-                  />
-                  <label for="adaptive-grid">Адаптивная</label>
+                  <label>
+                    <input
+                      class="filter_input filter_input_radio"
+                      type="radio"
+                      name="grid"
+                      checked
+                    />
+                    <span class="check_radio"></span>
+                    <span class="check_desc">Адаптивная</span>
+                  </label>
                 </li>
                 <li class="filter_option filter_radio">
-                  <input
-                    class="filter_input filter_input_radio"
-                    type="radio"
-                    name="grid"
-                    id="fix-grid"
-                  />
-                  <label for="fix-grid">Фиксированная</label>
+                  <label>
+                    <input
+                      class="filter_input filter_input_radio"
+                      type="radio"
+                      name="grid"
+                    />
+                    <span class="check_radio"></span>
+                    <span class="check_desc">Фиксированная</span>
+                  </label>
                 </li>
                 <li class="filter_option filter_radio">
-                  <input
-                    class="filter_input filter_input_radio"
-                    type="radio"
-                    name="grid"
-                    id="stretch-grid"
-                  />
-                  <label for="stretch-grid">Резиновая</label>
+                  <label>
+                    <input
+                      class="filter_input filter_input_radio"
+                      type="radio"
+                      name="grid"
+                    />
+                    <span class="check_radio"></span>
+                    <span class="check_desc">Резиновая</span>
+                  </label>
                 </li>
               </ul>
             </fieldset>
-            <fieldset>
-              <legend class="filter_title">особенности:</legend>
+            <fieldset class="features_filter">
+              <legend class="features_filter_title">особенности:</legend>
               <ul>
                 <li class="filter_option filter_checkbox">
-                  <input
-                    class="filter_input filter_input_checkbox"
-                    type="checkbox"
-                    name="features"
-                    id="slider"
-                    value="slider"
-                    checked
-                  />
-                  <label for="slider">Слайдер</label>
+                  <label>
+                    <input
+                      class="filter_input filter_input_checkbox"
```

Comment:
Незачем дублировать стили класса **visually-hidden**, можешь просто это добавить в верстке
```suggestion
                      class="filter_input filter_input_checkbox visually-hidden"
```